awpy/parser/demoparser.py

```diff
@@ -1023,17 +1028,23 @@ def _has_winner_and_not_winner(self, game_round: GameRound) -> bool:
             game_round["endCTScore"] == tie_score + 1
             and game_round["endTScore"] < tie_score
         )
-        # OT win scores are of the type:
-        # 15 + (4xN) with N a natural numbers (1, 2, 3, ...)
-        # So 19, 23, 27, ...
-        # So if you substract 15 from an OT winning round
-        # the number is divisible by 4
-        ot_valid_ct_win = (game_round["endCTScore"] - tie_score) % (
-            ot_tie_score + 1
-        ) == 0 and game_round["endTScore"] < game_round["endCTScore"]
-        ot_valid_t_win = (game_round["endTScore"] - tie_score) % (
-            ot_tie_score + 1
-        ) == 0 and game_round["endCTScore"] < game_round["endTScore"]
+        # OT draw scores are of the type
+        # 15 + 3xN with N a natural number(1, 2, 3, ...)
+        # So 18, 21, 24, 27
+        # Wins are 1 higher
+        # So 19, 22, 25, 28
+        # So if you substract 15 + 1 from an OT winning round
+        # the number is divisible by 3
+        ot_valid_ct_win = (
+            (game_round["endCTScore"] - tie_score - 1) % ot_tie_score == 0
+            and game_round["endTScore"] < (game_round["endCTScore"] - 1)
```

Comment:
Would it help to have a comment explaining why the difference has to be at least 2 because if one team reaches 18 then either they will reach 19 and win by at least two or the other team will also reach 18 and restart the overtime process essentially? Took me a second to realize why this section was needed initally and a small comment could help with that for others in the future if they have the same slight confusion

Reply:
Added one.